### PR TITLE
Clarify human review

### PR DESCRIPTION
**Pull request by wagnerand**

### CODE REVIEW

File: src/content/documentation/publish/what-does-review-rejection-mean-to-users.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ This article explains how users and people looking for your extension are affect
...
+Any extension you submit to [addons.mozilla.org](https://addons.mozilla.org) (AMO) is subject to two reviews. There is a machine validation of your extension as part of the submission flow, and a human review that can take place at any time, sometimes before publication.
 
-The machine validation tells you immediately if anything needs to be fixed to enable the publication of your extension. The human review occurs after publication and can occur at any time. When this review starts, the reviewer may ask for clarification about your extension. The outcome of the review could be the rejection of your extension’s latest version, and the rejection of earlier unreviewed versions if they also contain issues.
+The machine validation tells you immediately if anything needs to be fixed to enable the publication of your extension. The human review can occur at any time. In some cases, human review can be necessary for publication. When this review starts, the reviewer may ask for clarification about your extension. The outcome of the review could be the rejection of your extension’s latest version, and the rejection of earlier unreviewed versions if they also contain issues.
```
Comment (rebloor) [inline src/content/documentation/publish/what-does-review-rejection-mean-to-users.md:35]: I would remove the repetition, we just told the reader a human review can occur anytime. Also, can we clarify under what circumstances a human review before publication is necessary?
```suggestion
The machine validation tells you immediately if anything needs to be fixed to enable the publication of your extension. In some cases, a human review may be necessary for publication. When this review starts, the reviewer may ask for clarification about your extension. The outcome of the review could be the rejection of your extension’s latest version and the rejection of earlier unreviewed versions if they also contain issues.
```

Comment (wagnerand) [reply src/content/documentation/publish/what-does-review-rejection-mean-to-users.md:35]: The repetition was there in the original text as well, so I kept it.
We want to keep the current level of detail in the documentation.

Comment (rebloor) [reply src/content/documentation/publish/what-does-review-rejection-mean-to-users.md:35]: Indeed, but no reason why we cannot take this opportunity to make a small improvement :-)

Comment (wagnerand) [reply src/content/documentation/publish/what-does-review-rejection-mean-to-users.md]: Let's do that in a followup.

**Comment by wagnerand** [comment]
Can we merge this, please so it can be deployed soon?

**Review by rebloor — APPROVED** [review]
I would prefer if we remove the unnecessary duplication

**Comment by wagnerand** [comment]
> I would prefer if we remove the unnecessary duplication

No objection, feel free to update the patch, of merge as is and file a followup issue.

**Pull request merged**